src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx

```diff
@@ -107,4 +126,31 @@
   <data name="Downloading_IntelliSense_index_for_0" xml:space="preserve">
     <value>Downloading IntelliSense index for {0}</value>
   </data>
+  <data name="Regex_Comment" xml:space="preserve">
+    <value>Regex - Comment</value>
+  </data>
+  <data name="Regex_Character_class" xml:space="preserve">
+    <value>Regex - Character class</value>
```

Comment:
All other words are capitalized. Should "class" be as well?

Reply:
probably.  will fix.